ark/utils/load_utils.py

```diff
@@ -479,3 +483,92 @@ def load_tiled_img_data(data_dir, fovs, expected_fovs, channel, single_dir, file
                           dims=["fovs", "rows", "cols", "channels"])
 
     return img_xr
+
+
+def fov_to_ome(data_dir: Union[str, pathlib.Path], ome_save_dir: Union[str, pathlib.Path],
+               fovs: List[str] = None, channels: List[str] = None) -> None:
+    """
+    Converts a folder of FOVs into an OME-TIFF per FOV. This can be filtered out by
+    FOV and channel name.
+
+    Args:
+        data_dir (Union[str, pathlib.Path]):
+            Directory containing a folder of images for each the FOVs.
+        ome_save_dir (Union[str, pathlib.Path]):
+            The directory to save the OME-TIFF file to.
+        fovs (List[str]):
+            A list of FOVs to gather and save as an OME-TIFF file. Defaults to None
+            (Converts all FOVs in `data_dir` to OME-TIFFs).
+        channels (List[str], optional):
+            A list of channels to convert to an OME-TIFF. Defaults to None (Converts all channels
+            as channels in an OME-TIFF.)
+    """
+    iou.validate_paths([data_dir, ome_save_dir], data_prefix=False)
+
+    # Reorder the DataArray as OME-TIFFs require [Channel, Y, X]
+    fov_xr: xr.DataArray = load_imgs_from_tree(
```

Comment:
I think it'd be good to add `img_sub_folder` as an input to `load_imgs_from_tree` (and correspondingly add it as an argument to `fov_to_ome`)